### PR TITLE
Add Potugueseh translation [Resolves #221]

### LINKED ISSUE #221: Site Translation: Portuguese, Brazilian
Volunteers needed to complete this translation!
https://crowdin.com/project/ethereumfoundation/pt-BR#

We've been managing our efforts throught CrowdIn. Here's an invite to get started:
https://crowdin.com/project/ethereumfoundation/invite

Soon we'll have a public guide directly on the website (https://github.com/ethereum/ethereum-org-website/issues/318) but in the meantime, please leave a comment if you begin translating & we'll connect you with our Translation Program team lead to answer any questions you have about the process.

### PR DESCRIPTION
**Pull request by rodgeraraujo**
## Description
Add Portuguese translation at ethereum.org/br

## Related Issue
[#221](https://github.com/ethereum/ethereum-org-website/issues/221)

## Checklist:
- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/br/learn/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: Aprenda sobre Ethereum
+meta:
+  - name: description
+    content: A set of technical and non-technical articles, guides, and resources to learn about Ethereum.
```
Comment (samajammin) [inline docs/br/learn/index.md]: These descriptions aren't necessary but if we include them we should translate them.

File: docs/br/learn/index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  - property: og:title
+    content: Aprenda sobre Ethereum
+  - property: og:description
+    content: A set of technical and non-technical articles, guides, and resources to learn about Ethereum.
```
Comment (samajammin) [inline docs/br/learn/index.md]: Same here.

File: docs/br/learn/index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Um "contrato inteligente" é simplesmente um pedaço de código que está rodando no Ethereum.
+
+
+- Quer aprender como programar no Ethereum com contratos inteligentes? [ethereum.org/developers](/developers/)
+- [What is a Smart Contract?](https://github.com/ethereumbook/ethereumbook/blob/develop/07smart-contracts-solidity.asciidoc#what-is-a-smart-contract) *Nov 12, 2018 - Andreas M. Antonopoulos, Gavin Wood*
+- [What are Smart Contracts/Decentralized Applications?](https://docs.ethhub.io/ethereum-basics/what-is-ethereum/#what-are-smart-contracts-and-decentralized-applications) *Updated often - Ethhub*
+
+## Proof of Work and Mining
+
+Ethereum currently uses a system called “Proof of Work”. This allows the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain, and prevents certain kinds of economic attacks.
```
Comment (samajammin) [inline docs/br/learn/index.md]: This should also be translated.

File: docs/br/index.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    content: Ethereum é uma plataforma global e descentralizada de dinheiro e novos tipos de aplicativos. No Ethereum, você pode escrever um código que controla dinheiro e criar aplicativos acessíveis em qualquer lugar do mundo.
+  - property: og:title
+    content: Home | Ethereum
+  - property: og:description
+    content: Ethereum é uma plataforma global e descentralizada de dinheiro e novos tipos de aplicativos. No Ethereum, você pode escrever um código que controla dinheiro e criar aplicativos acessíveis em qualquer lugar do mundo.
+lang: pr-BR
+layout: home
+---
+
+<HomePage/>
```
Comment (samajammin) [inline docs/br/index.md]: In order to complete this homepage translation, we'll need to add the text & links in the translations.js file. See the existing sections as examples: https://github.com/ethereum/ethereum-org-website/blob/dev/docs/.vuepress/theme/utils/translations.js#L5

**Comment by rodgeraraujo** [comment]
@samajammin Ok! Later I complete the translations, and correct the remaining errors!

**Comment by samajammin** [comment]
Hey @rodgeraraujo - I'm sorry for the inconvenience but would you mind adding this translation to our CrowdIn project? This makes it easiest for our translation team to process. Here's an invite to get started:
https://crowdin.com/project/ethereumfoundation/invite

Here's the project:
https://crowdin.com/project/ethereumfoundation/sv-SE#

Once that's complete, I can fix these build errors & get this deployed. Feel free to post any questions you have here. I can connect you with the lead of our Translation Program to ensure you have everything you need.

Looking forward to getting your Portuguese translations up on the website!

**Comment by natacha-involves** [comment]
Hi, do you guys need help with this? I can help translate to Portuguese :)

**Comment by samajammin** [comment]
Thanks @natacha-involves! The more the merrier :)

Please see the CrowdIn invite above & let us know if you have any questions. It'd be great if you could help get @rodgeraraujo 's work into the Crowdin project.

**Comment by samajammin** [comment]
@rodgeraraujo @natacha-involves - apologies, looks like I shared the wrong link above. We actually have 2 separate Portuguese translations:
https://crowdin.com/project/ethereumfoundation/pt-PT#
https://crowdin.com/project/ethereumfoundation/pt-BR#
Please contribute to whichever translation you're more comfortable with. Thanks again!

**Comment by samajammin** [comment]
@rodgeraraujo if you don't mind granting me access to your fork of this repo, I can work to fix the failing build & complete this PR. Please let me know.

**Comment by samajammin** [comment]
@rodgeraraujo @natacha-involves could you please clarify which Portuguese this is? Brazilian?

It looks like that is now 100% in CrowdIn (🎉 ):
https://crowdin.com/project/ethereumfoundation/pt-BR#

The other Portuguese is at 60%:
https://crowdin.com/project/ethereumfoundation/pt-PT#

**Comment by samajammin** [comment]
@all-contributors please add @rodgeraraujo for translation

**Comment by samajammin** [comment]
@all-contributors please add @natacha-involves for translation

**Comment by samajammin** [comment]
Haven't heard from you in 20 days now, so closing this out for maintenance. Feel free to re-open & complete it! We've created a bounty for this work, which you can see here: #543

**Comment by samajammin** [comment]
Thank you again for the contributions!

**Pull request closed without merging**